webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -13,19 +20,29 @@ import { isSelecting } from '../../../util/strings'
 import Tooltip, {
   NORMAL_TOOLTIP_DELAY
 } from '../../../shared/components/tooltip/Tooltip'
+import { useDragAndDrop } from '../../../shared/hooks/useDragAndDrop'
+import { DragDropItemWithTarget } from '../../../shared/components/dragDrop/DragDropItemWithTarget'
 
 export interface ComparisonTableRowProps {
   path: string
   plots: ComparisonPlot[]
   nbColumns: number
   pinnedColumn: string
+  onLayoutChange: () => void
+  setOrder: (order: string[]) => void
+  order: string[]
+  bodyRef?: React.RefObject<HTMLTableSectionElement>
```

Comment:
[Q] Should this be `RefObject`? (I am asking because that is what I've been using)

Reply:
Copied and pasted without verifying. I'll fix it.